platforms/yttrex/shared/src/parser/parsers/home.ts

```diff
@@ -50,7 +50,7 @@ function dissectSelectedVideo(
   const e = ee as any;
 
   try {
-    infos.textTitle = e.querySelector('#video-title-link').textContent;
+    infos.textTitle = e.querySelector('#video-title-link')?.textContent;
```

Comment:
Sorry but is this the best way to fix it?

The title is mandatory, we need to have it. Also, there is more than one way to pick the title, look at the HTML:

![image](https://user-images.githubusercontent.com/89555/212682399-6ceacf9e-515a-42f5-9b9e-4f752ca96ed7.png)

The title can come from:

1. parsing `aria-label`
2. `textContent`
3. attribute `title`

I can understand the title might miss in certain conditions, but in that case the whole recommendation might also be discharged, don't you think?